### PR TITLE
Appendix: Keyboard shortcuts help

### PR DESCRIPTION
**Pull request by phw**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Picard provides many keyboard shortcuts, not all of them visible via the UI. This provides an documentation page listing the shortcuts.

### Description of the Change

Add a table to list all keyboard shortcuts available in the main window and also special keyboard shortcuts used by the script editor.

### Additional Action Required

There are some more specific UI elements that might implement additional shortcuts, such as the list editors used at various places or the tag editing dialog. Need to review those, but we could document them here as well.

### CODE REVIEW

File: appendices/keyboard_shortcuts.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.. MusicBrainz Picard Documentation Project
+
+.. |br| raw:: html
+
+   <br/>
```
Comment (phw) [inline appendices/keyboard_shortcuts.rst:5]: Not sure where to place this. Should probably be moved to a more general place.

Comment (rdswift) [reply appendices/keyboard_shortcuts.rst:5]: It seems to work okay where it is.  Let's leave it there for now.

**Comment by rdswift** [comment]
This looks good, but I have good news and bad news.  The bad news is that all the weird mac symbols completely break the LaTeX PDF writer in Sphinx.  The good news is that you didn't update `appendices/pdf_appendices.rst` to include this in the PDF output.  :wink: 

For now, let's just publish it this way (to the html and epub files only), and I'll try to find a way to generate the PDF file without LaTeX complaining about unmapped symbols.  I'll also update the French translation files once it is merged.

**Review by rdswift — APPROVED** [review]

**Comment by phw** [comment]
Interesting. As I did not have the Latex things installed I could not test this yet. We could use the commonly used names (⌃ = Control = Ctrl, ⌥ = Option = Opt = Alt, ⇧ = Shift, ⌘ = Command = Cmd) for the PDF case maybe. But not sure how to maybe translate this automatically.

For general view I think the symbols make sense. Mac users seem to used to them, it is what they see on their keyboards and how it is also presented to them if keyboard shortcuts are shown in the UI).

But sometimes it is only ⌘ and the other ones are written out (often with the short forms Ctrl, Opt).

We could also opt for using the names in general but maybe add a short reference to the symbols as Apple does it at the top of https://support.apple.com/en-us/HT201236 . That way we could use the table as is for both HTML and PDF view. We would then have this reference only being shown for the HTML view.

**Comment by phw** [comment]
Just a note: For the editor auto completion shortcut this already documents it for macOS as it will be in Picard 2.6.1, see https://github.com/metabrainz/picard/pull/1775 . Currently this shortcut in non-functional on macOS.

**Comment by rdswift** [comment]
> Interesting. As I did not have the Latex things installed I could not test this yet.

No problem.  I need to check on what versions I have installed, because according to the Sphinx documentation it should already use the `xelatex` engine by default which is supposed to properly handle UTF-8 input.  I think on my local Windows system it is using the `pdflatex` (or `latexpdf`) engine.

> We could use the commonly used names (⌃ = Control = Ctrl, ⌥ = Option = Opt = Alt, ⇧ = Shift, ⌘ = Command = Cmd) for the PDF case maybe. But not sure how to maybe translate this automatically.

I already have it done manually.  If I can't get Sphinx to build it properly with the symbols (my preferred solution), I'll put together a script similar to what we have for the tag mapping.

> For general view I think the symbols make sense. Mac users seem to used to them, it is what they see on their keyboards and how it is also presented to them if keyboard shortcuts are shown in the UI).

I agree.  They should be used if at all possible.  For sure on the HTML stuff because we know that works.

> We could also opt for using the names in general but maybe add a short reference to the symbols as Apple does it at the top of https://support.apple.com/en-us/HT201236 . That way we could use the table as is for both HTML and PDF view. We would then have this reference only being shown for the HTML view.

I still think that we should keep the symbols if at all possible.  We know that they work on the website and the epub documents, so it's just the pdf that I need to sort out.

**Comment by rdswift** [comment]
> Just a note: For the editor auto completion shortcut this already documents it for macOS as it will be in Picard 2.6.1, see [metabrainz/picard#1775](https://github.com/metabrainz/picard/pull/1775) . Currently this shortcut in non-functional on macOS.

Okay, but I think it's likely fine to publish this now.  If anyone notices the difference, it is easily explained.  :wink:

**Pull request merged**